include/crypto/modes.h

```diff
@@ -155,6 +155,12 @@ struct xts128_context {
     block128_f block1, block2;
 };
 
+/* XTS mode for SM4 algorithm specified by GB/T 17964-2021 */
+int CRYPTO_xts128gb_encrypt(const XTS128_CONTEXT *ctx,
+                          const unsigned char iv[16],
+                          const unsigned char *inp, unsigned char *out,
+                          size_t len, int enc);
+
```

Comment:
Please rename the function to ossl_crypto_xts128gb_encrypt(). Also please fix the indentation below.

Reply:
fixed in [26d03dc](https://github.com/openssl/openssl/commit/26d03dccfaa9e801e55846f7ac213c3dcdabbc52)